bot.py

```diff
@@ -558,6 +558,11 @@ async def sendEmbed(self, ctx, townInfo):
 class Gameplay(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
+        self.currentlyActiveGames = {}
+        #self.cleanupInactiveGames.start() This should only be run if we have active games in the DB
```

Comment:
Currently without this line the timer only gets started if you call !endgame which feels backwards

Reply:
It should only get called when you call currgame, not endgame.

I tested by calling ~night and waiting (with shorter timeouts for debugging). It correctly called the timer function in response to ~night.